README.md

```diff
@@ -97,51 +72,153 @@ dependencies {
+- [context-aware](https://docs.openfeature.dev/docs/reference/concepts/evaluation-context) evaluation
+
+## 🚀 Usage:
+
+### Basics:
+
 ```java
-class MyApp {
-    public void example(){
-        OpenFeatureAPI api = OpenFeatureAPI.getInstance();
-        api.setProvider(new NoOpProvider());
-        Client client = api.getClient();
-        // Now use your `client` instance to evaluate some feature flags!
-    }
+public void example(){
+
+    // configure a provider
+    OpenFeatureAPI api = OpenFeatureAPI.getInstance();
+    api.setProvider(new MyProviderOfChoice());
+
+    // create a client
+    Client client = api.getClient();
+
+    // get a bool flag value
+    boolean flagValue = client.getBooleanValue("boolFlag", false);
 }
 ```
-## Contacting us
-We hold regular meetings which you can see [here](https://github.com/open-feature/community/#meetings-and-events).
 
-We are also present on the `#openfeature` channel in the [CNCF slack](https://slack.cncf.io/).
+### Context-aware evaluation:
 
-## Developing
+Sometimes the value of a flag must take into account some dynamic criteria about the application or user, such as the user location, IP, email address, or the location of the server.
+In OpenFeature, we refer to this as [`targeting`](https://docs.openfeature.dev/specification/glossary#targeting).
+If the flag system you're using supports targeting, you can provide the input data using the `EvaluationContext`.
 
-### Integration tests
+```java
+// global context for static data
+OpenFeatureAPI api = OpenFeatureAPI.getInstance();
+Map<String, Value> attributes = new HashMap<>();
+attributes.put("appVersion", new Value(System.getEnv("APP_VERSION")));
+EvaluationContext apiCtx = new ImmutableContext(attributes);
+api.setEvaluationContext(apiCtx);
+
+// request context
+Map<String, Value> attributes = new HashMap<>();
+attributes.put("email", new Value(session.getAttribute("email")));
+attributes.put("product", new Value(productId));
+String targetingKey = session.getId();
+EvaluationContext reqCtx = new ImmutableContext(targetingKey, attributes);
+
+// use merged contextual data to determine a flag value
+boolean flagValue = client.getBooleanValue("some-flag", false, reqCtx);
+```
 
-The continuous integration runs a set of [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features/evaluation.feature) using [`flagd`](https://github.com/open-feature/flagd). These tests do not run with the default maven profile. If you'd like to run them locally, you can start the flagd testbed with
+### Providers:
 
+To develop a provider, you need to create a new project and include the OpenFeature SDK as a dependency. This can be a new repository or included in an existing contrib repository available under the OpenFeature organization. Finally, you’ll then need to write the provider itself. In most languages, this can be accomplished by implementing the provider interface exported by the OpenFeature SDK.
```

Comment:
```suggestion
To develop a provider, you need to create a new project and include the OpenFeature SDK as a dependency. This can be a new repository or included in [the existing contrib repository](https://github.com/open-feature/java-sdk-contrib) available under the OpenFeature organization. Finally, you’ll then need to write the provider itself. This can be accomplished by implementing the `FeatureProvider` interface exported by the OpenFeature SDK.
```

Reply:
I'll add this to the template.